features/cart/shopping_cart/removing_cart_item.feature

```diff
@@ -7,10 +7,29 @@ Feature: Removing cart item from cart
     Background:
         Given the store operates on a single channel in "France"
         And the store has a product "T-shirt banana" priced at "€12.54"
-        And I add this product to the cart
 
     @ui
     Scenario: Removing cart item
-        Given I see the summary of my cart
-        When I remove product "T-shirt banana" from the cart
+        Given I add this product to the cart
+        When I see the summary of my cart
+        And I remove product "T-shirt banana" from the cart
+        Then my cart should be empty
+        And my cart's total should be "€0.00"
+
+    @ui
+    Scenario: Removing cart item when the store has defined default shipping method
+        Given the store has "UPS" shipping method with "€20.00" fee
+        And I add this product to the cart
+        When I see the summary of my cart
+        And I remove product "T-shirt banana" from the cart
         Then my cart should be empty
+        And my cart's total should be "€0.00"
+
+    @ui
+    Scenario: Checking cart's total after removing one item
+        Given I add this product to the cart
```

Comment:
Same step in every scenario, wouldn't it be better to move it to the background?